graphkb/genes.py

```diff
@@ -349,12 +349,13 @@ def get_pharmacogenomic_info(
                 "conditions.reference2.biotype",
                 "conditions.reference2.displayName",
                 "source.name",
+                "source.displayName",
             ],
         },
         ignore_cache=False,
     ):
         if record["source"]:  # type: ignore
-            if record["source"]["name"].lower() in PHARMACOGENOMIC_SOURCE_EXCLUDE_LIST:  # type: ignore
+            if record["source"]["displayName"] in GSC_PHARMACOGENOMIC_SOURCE_DISPLAYNAME_EXCLUDE_LIST:  # type: ignore
```

Comment:
can we make the exclude list passable as a parameter?

Reply:
Sure, done.
But let's not worry too much about this, given Yaoqin's statements on the ticket, about using other methods to select.